language-server/test/dotty/tools/languageserver/CompletionTest.scala

```diff
@@ -842,4 +842,10 @@ class CompletionTest {
           |object Main { "abc".xx${m1} }""".withSource
       .completion(m1, Set())
   }
+
+  @Test def wrongAnyMember: Unit = {
+    code"""|import scala.util.chaining.`sc${m1}
```

Comment:
It would be good to test some cases with more natural usages of backticks - if the name clashes with a keyword or contains spaces or would be normally disallowed for some other reason

Reply:
I added some more test cases, let me know if you have any other ideas!